src/domain/WinningLotto.js

```diff
@@ -0,0 +1,74 @@
+import { ErrorLotto } from "../constants/error";
+import {
+  LOTTO_NUMBER_LENGTH,
+  MAX_NUMBER,
+  MIN_NUMBER,
+} from "../constants/number";
+
+export const FIRST_PRIZE = 2000000000;
+export const SECOND_PRIZE = 30000000;
+export const THIRD_PRIZE = 1500000;
+export const FOURTH_PRIZE = 50000;
+export const FIFTH_PRIZE = 5000;
+
+export const PRIZE = {
+  FIRST: { rank: "FIRST", matchCount: 6, prize: FIRST_PRIZE, isBonus: false },
+  SECOND: { rank: "SECOND", matchCount: 5, prize: SECOND_PRIZE, isBonus: true },
+  THIRD: { rank: "THIRD", matchCount: 5, prize: THIRD_PRIZE, isBonus: false },
+  FOURTH: {
+    rank: "FOURTH",
+    matchCount: 4,
+    prize: FOURTH_PRIZE,
+    isBonus: false,
```

Comment:
```suggestion
    isBonus: false,
```
`is`라는 prefix는 이런 모습에 더 적합하지 않을까요?

```js
function isBonus(lotto, lottoNumber) {
  return !lotto.contains(lottoNumber)
}
```